# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/JSScript.cpp
# Commit: 2bc5949902ac
# Full Hash: 2bc5949902acc042205fc7e85260bbbc37dbedc7
# Author: Matthew Gaudet <mgaudet@mozilla.com>
# Date: 2021-09-18 09:46:57
# Regressor Bug: 1688788
# File Overlap Count: 1
# Description:
#   Bug 1688788 - Convert ScriptPreloader to manage Stencils, not Scripts. r=kmag,tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113013
# ==============================================================================

diff -r e660e2c3c3d1 -r 2bc5949902ac js/src/vm/JSScript.cpp
--- a/js/src/vm/JSScript.cpp	Sat Sep 18 01:29:45 2021 +0000
+++ b/js/src/vm/JSScript.cpp	Sat Sep 18 01:29:45 2021 +0000
@@ -1122,7 +1122,7 @@
 // FillImmutableFlagsFromCompileOptionsForTopLevel above.
 //
 // If isMultiDecode is true, this check minimal set of CompileOptions that is
-// shared across multiple scripts in JS::DecodeMultiOffThreadScripts.
+// shared across multiple scripts in JS::DecodeMultiOffThreadStencils.
 // Other options should be checked when getting the decoded script from the
 // cache.
 bool js::CheckCompileOptionsMatch(const ReadOnlyCompileOptions& options,
@@ -1207,7 +1207,7 @@
     // should fail. This only applies to the top-level and not its inner
     // functions.
     //
-    // Also, JS::DecodeMultiOffThreadScripts uses single CompileOptions for
+    // Also, JS::DecodeMultiOffThreadStencils uses single CompileOptions for
     // multiple scripts with different CompileOptions.
     // We should check minimal set of common flags here, and let the consumer
     // check the full flags when getting from the cache.